dockerfiles/Dockerfile.tensorrt

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with TensorRT integration
+
+# nVidia TensorRT Base Image
+FROM nvcr.io/nvidia/tensorrt:19.03-py3
```

Comment:
replace with FROM nvcr.io/nvidia/tensorrt:19.02-py3

TensorRT 19.03 image is based on TensorRT 5.1
I think it would be better to use TensorRT image 19.02 because ORT source is still using the TRT 5.0 branch of onnx-tensorrt.

Reply:
Updated.